test/suite/test_layered08.py

```diff
@@ -80,10 +79,8 @@ def test_layered_read_write(self):
         # Inserted timing delays around reopen, apparently needed because of the
         # layered table watcher implementation
         import time
-        time.sleep(1.0)
         follower_config = self.conn_base_config + 'disaggregated=(role="follower")'
         self.reopen_conn(config=follower_config)
-        time.sleep(1.0)
```

Comment:
Just one question why remove these sleeps?

Reply:
The sleeps were originally put in because the first version of layered tables needed them to function.  They haven't been needed for a long time, so it's time to clean them up (and make the tests run faster).